site/content/user-guide/device-registry.md

```diff
@@ -3,17 +3,38 @@ title = "Device Registry"
 weight = 205
 +++
 
+The Device Registry component is an example implementation of the following Hono APIs:
+
+* [Tenant API]({{< relref "api/Tenant-API.md" >}})
+* [Device Registration API]({{< relref "api/Device-Registration-API.md" >}})
+* [Credentials API]({{< relref "api/Credentials-API.md" >}})
+
+It implements AMQP 1.0 API endpoints and provides JSON based file persistence for data. 
+
 In addition to the AMQP 1.0 based API endpoints, the Device Registry also exposes HTTP resources for managing the contents of the registry.
 
-<!--more-->
-Please note that the HTTP resources exposed by the Device Registry are **not** part of the *official* definition of the Device Registration, Credentials and Tenant APIs.
- 
 {{% warning %}}
 The Device Registry is not intended to be used in production environments. In particular, access to the HTTP resources described below is not restricted to authorized clients only.
 
 The resources have been designed to provide convenient access to the registry's content using command line tools like *curl* or *HTTPie*.
 {{% /warning %}}
 
+## Implementation details
+
+This section describes implementation specifics of certain parts of the Device Registry.
+
+### Get Credentials
+
+The [Get Credentials]({{< relref "api/Credentials-API.md#get-credentials" >}}) operation of Credentials API defines that arbitrary array of properties can be passed to help identify the device. 
```

Comment:
FMPOV this whole section should go down to the description of the *Get Credentials* resource. It feels very much out of context in its current place.